api/mapping/forms.py

```diff
@@ -37,10 +43,15 @@ class ScanReportForm(forms.Form):
         queryset=Dataset.objects.order_by("name"),
         widget=forms.Select(attrs={"class": "form-control"}),
     )
+    visibility = forms.ChoiceField(
```

Comment:
One last comment: does this belong in this PR? Seems like it should be in #403 (which will be replacing it with React anyway)

Reply:
Yeah I only needed it at the time for Scan Report upload to work, so I rebased on that branch, I'll remove it from this PR